src/initialization/FGInitialCondition.cpp

```diff
@@ -1030,18 +1030,21 @@ bool FGInitialCondition::Load(const SGPath& rstfile, bool useStoredPath)
   double version = HUGE_VAL;
```

Comment:
```suggestion
```
The declaration of the variable `version` should now be moved in the `if` block.